operator/v1/types_ingress.go

```diff
@@ -1306,6 +1306,23 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Format=duration
 	// +optional
 	TLSInspectDelay *metav1.Duration `json:"tlsInspectDelay,omitempty"`
+
+	// healthCheckInterval defines how long the router waits between two consecutive
+	// health checks on its configured backends.  This value is applied globally as
+	// a default for all routes, but may be overridden per-route by the route annotation
+	// "router.openshift.io/haproxy.health.check.interval".
+	//
+	// Setting this to less than 5s can cause excess traffic due to too frequent
+	// TCP health checks and accompanying SYN packet storms.  Alternatively, setting
+	// this too high can result in increased latency, due to backend servers that are no
+	// longer available, but haven't yet been detected as such.
+	//
+	// If unset, the default healthCheckInterval is 5s.
```

Comment:
Is there any reason not to set a default tag so that this is defaulted by OpenAPI when omitted? Else I expect you'll have controllers doing the defaulting right? 

```
// + kubebuilder:Default:="5s"
```

Reply:
I'd also be ok with "The platform chooses a default, which is subject to change over time. The current default is 5s"

Reply:
We don't currently have any plans to make this change.  HAProxy has a default of 2s, which we currently set to 5s.

Reply:
Updated, as discussed.

Reply:
> Is there any reason not to set a default tag so that this is defaulted by OpenAPI when omitted?

Defaulting doesn't work with duration values.